apps/src/code-studio/pd/application_dashboard/cohort_view_table.jsx

```diff
@@ -255,7 +255,7 @@ export class CohortViewTable extends React.Component {
   };
 
   // Format dates as abbreviated month and day, e.g. "Mar 9"
-  formatDate = (iso8601Date) => moment(iso8601Date).format("MMM D");
+  formatDate = (iso8601Date) => iso8601Date ? moment(iso8601Date).format("MMM D") : "";
```

Comment:
Are you hoping to see if iso8601 exists, is defined, is truthy?

Reply:
I'm hoping to see if it exists. That parameter is the `date_accepted` value for the application, and when it's nil we were displaying "Invalid date"

Reply:
Then I suggest explicitly saying `iso8601Date !== null`.  `''` evaluatees to false which may throw things off

Reply:
So I guess I am actually checking to see if `iso8601Date` is truthy